core/src/tasks/profile/enqueueExports.ts

```diff
@@ -0,0 +1,49 @@
+import { Op } from "sequelize";
+import { Profile } from "../../models/Profile";
+import { CLSTask } from "../../classes/tasks/clsTask";
+import { plugin } from "../../modules/plugin";
+import { Import } from "../../models/Import";
+import { CLS } from "../../modules/cls";
+
+export class ProfilesEnqueueExports extends CLSTask {
+  constructor() {
+    super();
+    this.name = "profiles:enqueueExports";
+    this.description =
+      "If a profile is ready and has an import that has not exported, enqueue profile:export for them";
```

Comment:
Is it possible for a profile to be ready, but for its properties not to be ready?  It shouldn't be, but we should confirm this... otherwise we can add that check here as well.

Reply:
No, profiles are only marked ready by `profiles:checkReady` when all their properties are ready.